Microsoft.SystemForCrossDomainIdentityManagement/Protocol/PatchOperation2CombinedDeserializingFactory.cs

```diff
@@ -0,0 +1,87 @@
+﻿//------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+//------------------------------------------------------------
+
+namespace Microsoft.SCIM.Protocol
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Runtime.Serialization;
+    using System.Runtime.Serialization.Json;
+    using System.Threading;
+
+    class PatchOperation2CombinedDeserializingFactory :
+        ProtocolJsonDeserializingFactory<PatchOperation2Combined>
+    {
+
+        private static readonly Lazy<DataContractJsonSerializer> JsonSerializer =
+            new Lazy<DataContractJsonSerializer>(
+                () =>
+                    new DataContractJsonSerializer(
+                        typeof(PatchOperation2Combined)));
+
+        public override PatchOperation2Combined Create(IReadOnlyDictionary<string, object> json)
+        {
+            if (null == json)
+            {
+                throw new ArgumentNullException(nameof(json));
+            }
+
+            IReadOnlyDictionary<string, object> normalizedJson = this.Normalize(json);
+            string serialized = JsonFactory.Instance.Create(normalizedJson, this.AcceptLargeObjects);
+
+            MemoryStream stream = null;
+            try
+            {
+                stream = new MemoryStream();
+                Stream streamed = stream;
+                StreamWriter writer = null;
+                try
+                {
+                    writer = new StreamWriter(stream);
+                    stream = null;
+                    writer.Write(serialized);
```

Comment:
All the logic in this file is the same as the base method. The logic to set the correct attributes with the correct values should be set in this method.